packages/core/src/components/time-picker-rework/time-picker-rework.scss

```diff
@@ -0,0 +1,116 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import 'common-variables';
+@import 'mixins/shadow-dom/component';
+@import 'components/form/input';
+
+:host {
+  @include ix-component;
+  @import 'legacy/components/forms';
+
+  display: block;
```

Comment:
Missing min-width and width which results in a full screen timepicker:
![image](https://github.com/siemens/ix/assets/2144313/4f27b7c4-e4c9-448d-882f-d0a305fa8f05)

Reply:
We should align it to date/datetime-picker sizing